web/src/app/modules/navigation/modules/navigation-bar/components/navigation-bar.component.html

```diff
@@ -12,6 +12,7 @@
     <span class="navbar-text">
         {{'version' | translate}}:&nbsp;{{version}}&nbsp;
         <operation-monitor></operation-monitor>&nbsp;
+        <themer style="display: inline-block;"></themer>&nbsp;

```

Comment:
Inline styles are generally a bad practice. Could you use a css class here instead?

Reply:
Yes, I can. The reason I used an inline style was that, this is the only place this style is required. I will change it and push again.

Reply:
Refactored and pushed.